### PR TITLE
Schutzbot: Start building on aarch64 for RHEL/CentOS

### PR DESCRIPTION
**Pull request by atodorov**
This pull request includes:

- [x] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change

### CODE REVIEW

**Comment by atodorov** [comment]
@major IDK if the slave systems will be provisioned automatically or if you need to make some adjustments to our AWS account. Let's see how this goes.

**Review by teg — COMMENTED** [review]
Not sure we have the required workers set up (at least not in PSI), but if we can, this would be good to have!

**Comment by ondrejbudai** [comment]
AFAIK, PSI doesn't have support for aarch64. It's possible though to test CS on AWS, I just need to add an image definition to the Jenkins settings. I will ping you when it's done.

**Comment by atodorov** [comment]
> AFAIK, PSI doesn't have support for aarch64. 

I think we can use an AWS host for RHEL + aarch64 and limit the types of testing we do there / targets we build against!


> It's possible though to test CS on AWS, I just need to add an image definition to the Jenkins settings. I will ping you when it's done.

I can probably do that myself, let me know where to look.

**Comment by ondrejbudai** [comment]
> I think we can use an AWS host for RHEL + aarch64 and limit the types of testing we do there / targets we build against!

However, we currently cannot access the internal repos (and thus RHEL 8.4) from AWS.

**Comment by atodorov** [comment]
> 
> However, we currently cannot access the internal repos (and thus RHEL 8.4) from AWS.

Yes but we can only build & test on released RHEL 8 like we already do. This is still better than not testing aarch64 at all.

**Comment by ondrejbudai** [comment]
Ah, yeah, that's definitely possible. I think many people are interested in enabling testing for more distros - I will try to prepare a guide. :)

**Comment by ondrejbudai** [comment]
I added these AMI instance types to Jenkins config:

```
rhel8cloudbase && aarch64 && aws
cs8cloudbase && aarch64 && aws
```

As I said, I don't think that can currently test RHEL 8.4 on aarch64 but at least stable RHEL and Centos Stream should be possible.

**Comment by ondrejbudai** [comment]
Hmm, the tests seem to be broken right now. Alex, can you please take a look?

File: schutzbot/Jenkinsfile [hunk 1]
```diff
@@ -675,36 +675,6 @@ pipeline {
                         }
                     }
                 }
-                stage('EL8 aarch64 Image') {
```
Comment (teg) [inline schutzbot/Jenkinsfile]: Should this be part of this patch?

Comment (atodorov) [reply schutzbot/Jenkinsfile]: Nope, squashed into the previous commit.

**Comment by atodorov** [comment]
> Hmm, the tests seem to be broken right now. Alex, can you please take a look?

There's a failure trying to download a nightly repo with dnf. The reason this happens is because a file check 
`-f rhel8nightly.repo` succeeds, which happens because a previous step manages to somehow unstash files with the same name: `Restore files previously stashed` which are not supposed to exist IMO. Still investigating.

**Comment by atodorov** [comment]
Merged as #1287

**Pull request closed without merging**